packages/wrangler/package.json

```diff
@@ -119,11 +121,12 @@
 		"Cloudflare_CA.pem"
 	],
 	"scripts": {
-		"clean": "rm -rf wrangler-dist miniflare-dist",
+		"clean": "rm -rf wrangler-dist miniflare-dist emitted-types tsconfig.emit.tsbuildinfo",
```

Comment:
why do you delete the cache file here?

Reply:
i added this because it wasn't running at all when i would modify the extraction script but not the source ts, but i think you're right -- the usual use case of modifying wrangler but not the extraction script would benefit from keeping it around